packages/links/links.0.7.3/opam

```diff
@@ -0,0 +1,68 @@
+opam-version: "1.2"
+maintainer: "Simon Fowler <simon.fowler@ed.ac.uk>"
+authors: "The Links Team <links-dev@inf.ed.ac.uk>"
+homepage: "https://github.com/links-lang/links"
+dev-repo: "https://github.com/links-lang/links.git"
+bug-reports: "https://github.com/links-lang/links/issues"
+license: "GPL-2"
+
+available: [
+  ocaml-version >= "4.06.0"
+]
+
+build: [
+  [ "jbuilder" "subst" "-n" name ] {pinned}
+  [ "jbuilder" "build" "-p" name "-j" jobs ]
+]
+
+install: [
```

Comment:
Have you not found any ways of installing those files using `jbuilder`'s `install` stanza ?

Reply:
We are not aware of any way to copy those files and directories programmatically using jbuilder's stanzas.

Reply:
/cc @diml and @rgrinberg

Reply:
Currently we can't put globs in `install` stanzas. As long as these don't include generated files you could either use the promote feature to maintain a generated and committed jbuild file, or use [the OCaml syntax](https://dune.readthedocs.io/en/latest/jbuild.html#id4).

If not done already, please open a ticket on ocaml/dune about this with your use case.